tools/test-proxy/Azure.Sdk.Tools.TestProxy/Properties/launchSettings.json

```diff
@@ -17,5 +9,13 @@
       },
       "applicationUrl": "https://localhost:5001;http://localhost:5000"
     }
+  },
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:5001/",
+      "sslPort": 44362
+    }
```

Comment:
@scbedd, I'm not sure why this is in there, I haven't touched the property settings.